src/main/scala/zio/prelude/NonEmptyList.scala

```diff
@@ -481,7 +546,16 @@ object NonEmptyList extends LowPriorityNonEmptyListImplicits {
     }
 
   /**
-   * Constructs a `NonEmptyList` from an `Iterable`.
+   * Constructs a `NonEmptyList` from an `Iterable` or `None` otherwise.
+   */
+  def fromIterable[A](iterable: Iterable[A]): Option[NonEmptyList[A]] =
```

Comment:
Maybe `fromIterableOption` to avoid overloading?

Reply:
Done.